test/promql/tests/test_network.yaml

```diff
@@ -0,0 +1,123 @@
+# This is the main input for unit testing.
+# Only this file is passed as command line argument.
+
+rule_files:
+  - ../extracted-rules.yaml
+
+evaluation_interval: 30s
+
+tests:
+  - interval: 30s
```

Comment:
If we already have readability comments, I'd add a short comment about each value in a series being a result of a 30s interval

Reply:
Done